README.md

```diff
@@ -324,8 +324,10 @@ Example: `./profiler.sh -t 8983`
 * `--title TITLE`, `--width PX`, `--height PX`, `--minwidth PX`, `--reverse` - FlameGraph parameters.  
 Example: `./profiler.sh -f profile.svg --title "Sample CPU profile" --minwidth 0.5 8983`
 
-* `-f FILENAME` - the file name to dump the profile information to.  
-Example: `./profiler.sh -o collapsed -f /tmp/traces.txt 8983`
+* `-f FILENAME` - the file name to dump the profile information to.
+`%p` in the file name is expanded to the PID of the target JVM;
+`%t` — to the timestamp.
```

Comment:
It might be worth clarifying at which time the time stamp is recorded. Is that correct that the timestamp corresponds to the time the command is received (i.e., if file option is passed with the start command, then it corresponds to the start time, if with the stop — then to the stop time)? If that's correct, how'd you put it in a concise way?

Reply:
You are right, '%t' is expanded when the profiler parses arguments, i.e. at the time of command execution.